content/en/catalyst/fund8/challenges/dapps-and-integrations/campaign-brief.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "DApps and Integrations"
+linkTitle: "DApps and Integrations"
+type: docs
+hide_feedback: true
+description: "What applications will provide the most value for end users in 2022?"
+---
+Fund Size: **2,500,000 USD paid in Ada**
+
+[Campaign page](https://cardano.ideascale.com/c/idea/383335) and [browse ideas]().
+
+## Campaign Brief
+What applications will provide the most value for end users in 2022?
+
+### Why is it important?
+Quality Dapps and integrations will increase utility & adoption of Cardano. A broad funding challenge like this encourages more proposals.
+
+### What does success look like?
+Novel and viable applications get built of top of Cardano attracting users. Integrations manage to pull in more adoption.
+
+### Key Metrics to measure
+To assess the challenge we will be asking ourselves: Did our efforts manage to attract users to the platform? Did our efforts result in new, viable applications built on top of Cardano?
+
+Let an "application output" and an "application transaction" be an output and transactions that use the applications' script addresses, and an "application token" be a token that's issued by the application.
+
+### Usage of applications developed during the challenge:
+
+- Number/growth of application output
+- Number/growth of application transactions
+- The total volume of Ada involved in application transactions
+- Total/growth of resource usage by applications
+- Number/growth of distinct addresses mentioned in application transactions
+
+### If the application uses tokens:
+
+- Number/growth of application tokens
+- Number/growth of distinct addresses holding application tokens
+- Viability of applications developed during the challenge:
+- How many of the applications are still being developed at the end of 2022?
+- How many of the applications made a profit for their creators?
+- How many of the teams which built the applications grew during 2022?
+- How many of the teams which built the applications built (or plan to build) another application?
+
+## Challenge Brief
+Cardano is an ecosystem that sets itself apart through its rigorous research and strives to do things correctly. We expect all dApp projects applying for funding to aspire to the highest possible levels of certification. To understand what this refers to and practically means, please watch Charles Hoskinson's video on this topic: "DApp certification and Community Curation", https://youtu.be/Y7zcYSQarI0.  
```

Comment:
@PardusEidolon can you turn this into a Hugo Youtube shortcode to embed the video correctly.

Reply:
:+1: